include/sprite.h

```diff
@@ -263,6 +272,13 @@ void sub_8003914(Sprite *);
 void sub_80047A0(u16, s16, s16, u16);
 
 s16 sub_8004418(s16 x, s16 y);
+void numToASCII(u8 digits[5], u16 number);
+
+#if ((GAME == GAME_SA1) || (GAME == GAME_SA2))
+#define GET_SPRITE_ANIM(s) ((s)->graphics.anim)
```

Comment:
So definitely not in graphics in SA3?

Reply:
In SA3 `Sprite` just doesn't have the `Graphics` struct integrated, which makes more sense, imo.
`typedef struct {
    /* 0x00 */ u8 *tiles; // in VRAM
    /* 0x04 */ u32 frameNum;
    /* 0x08 */ u32 frameFlags; 
    /* 0x0C */ u16 anim;
    /* 0x0E */ u16 animCursor;
    /* 0x10 */ s16 x;
    /* 0x12 */ s16 y;
    /* 0x14 */ s16 oamFlags;
    /* 0x16 */ s16 qAnimDelay;
    /* 0x18 */ u16 prevAnim;
    /* 0x1A */ u8 variant;
    /* 0x1B */ u8 prevVariant;
    /* 0x1C */ u8 animSpeed;
    /* 0x1D */ u8 oamBaseIndex;
    /* 0x1E */ u8 numSubFrames;
    /* 0x1F */ u8 palId; // (0 - 15)
    /* 0x20 */ Hitbox hitboxes[1];
} Sprite; /* size: 0x28 `

Reply:
How does it copy the graphics?